Code/Tools/ProjectManager/Source/Application.cpp

```diff
@@ -41,6 +40,8 @@ namespace O3DE::ProjectManager
         QCoreApplication::setApplicationName(applicationName);
         QCoreApplication::setApplicationVersion("1.0");
 
+        SetupCrashHandler();
```

Comment:
It looks like this will not build on non windows platforms. Did you forget to add the AZ_TRAIT_PROJECT_MANAGER_CRASH_HANDLER here so this isn't called on other platforms?